Library/Formula/ice.rb

```diff
@@ -18,6 +22,36 @@ class Ice < Formula
   patch :DATA
 
   def install
+    resource("berkeley-db").stage do
+
+      # Fix build under Xcode 4.6
+      # Double-underscore names are reserved, and __atomic_compare_exchange is now
+      # a built-in, so rename this to something non-conflicting.
+      inreplace "src/dbinc/atomic.h" do |s|
+        s.gsub! "__atomic_compare_exchange", "__atomic_compare_exchange_db"
+      end
+
+      # BerkeleyDB dislikes parallel builds
+      ENV.deparallelize
+
+      # --enable-compat185 is necessary because our build shadows
+      # the system berkeley db 1.x
+      args = %W[
+        --disable-debug
+        --prefix=#{libexec}
+        --mandir=#{man}
```

Comment:
` --mandir=#{libexec}/man ` would be good here, otherwise the manpages may get linked into the main ` HOMEBREW_PREFIX ` which would clash with the new BDB.